cmd/policy-main.go

```diff
@@ -434,23 +447,28 @@ func runPolicyCmd(args cli.Args) {
 }
 
 func mainPolicy(ctx *cli.Context) error {
-
 	// check 'policy' cli arguments.
 	checkPolicySyntax(ctx)
 
 	// Additional command speific theme customization.
 	console.SetColor("Policy", color.New(color.FgGreen, color.Bold))
 
 	switch ctx.Args().First() {
+	case "set", "set-json", "get", "get-json":
+		// policy set accesspermission alias/bucket/prefix
+		// policy set-json path-to-policy-json-file alias/bucket/prefix
+		// policy get alias/bucket/prefix
+		// policy get-json alias/bucket/prefix
```

Comment:
are these commented lines left here intentionally?

Reply:
yes. the `master branch` does have sample command in comment which i followed.